blueprint/src/chapter/pfr-entropy.tex

```diff
@@ -241,9 +241,9 @@ \subsection{Endgame}
   \uses{first-estimate, second-estimate}
```

Comment:
Should we remove this dependence on `first-estimate`, since it ends up not being used in the proof?